src/fonts/index.scss

```diff
@@ -0,0 +1,3 @@
+@import "museo";
+// @import "open-sans";
```

Comment:
I had left it out temporarily, as I didn't know if all of the additional character sets (Cyrillic, Greek) were necessary. I'll uncomment this.